### ISSUE TITLE
Apply git-flow for the project 

### ISSUE THREAD

**Issue by volkodavs** [matchbook-technology/matchbook-sdk#62]
We should apply git branching workflow for the `matchbook-sdk` project  

https://jeffkreeftmeijer.com/git-flow/

**Comment by alessandrodalbello** [comment]
Can we also review/rediscuss issue templates at this stage?

**Comment by volkodavs** [comment]
sure we can! What's in your mind about issue template?

**Comment by alessandrodalbello** [comment]
I was thinking to investigate templates of other open-source projects (to get inspired :slightly_smiling_face:), using headers instead of bold text for sections (as I did in recent issues I opened) and reword the descriptions a bit.

**Comment by volkodavs** [comment]
yeah, let's have a meeting about it. At the moment we're using default github issues template
